auth-store/index.js

```diff
@@ -0,0 +1,45 @@
+import { createStore } from '@logux/state'
+
+export function Auth(client) {
```

Comment:
Builders accept `id` as a first argument. `useSync` will not work with `Auth`.

Maybe we should call it `createAuth()` to avoid confusion with `SyncMap` builders?

Reply:
I'm a little confused about naming. I have to remember that the capital letter is for Builders 🤦🏻‍♀️